runtime/src/locked_accounts_results.rs

```diff
@@ -30,6 +33,10 @@ impl<'a, 'b> LockedAccountsResults<'a, 'b> {
     pub fn transactions(&self) -> &[Transaction] {
         self.transactions
     }
+
+    pub fn txs_iteration_order(&self) -> Option<&[usize]> {
+        self.txs_iteration_order.as_ref().map(|v| v.as_slice())
```

Comment:
@mvines Why do we need this wrapper?

Reply:
I was just following the pattern for this struct.  I'm not a big fan of this style generally, but I do see the utility of abstracting implementation details.